tests/core/go_binary/BUILD.bazel

```diff
@@ -19,5 +19,14 @@ go_binary(
     srcs = ["custom_bin.go"],
     out = "alt_bin",
 )
-    
+
+go_binary(
+    name = "goos_pure_bin",
+    srcs = [
+        "broken_cgo.go",
```

Comment:
It looks like this file is still being included in the build. I don't believe anything will filter out the `_cgo.go` suffix; that only works for goos / goarch suffixes. Instead, could you add `// +build cgo` to the top of the file?

Reply:
Thanks for the suggestion. Looks like the tests are passing now.